queries/java/indents.scm

```diff
@@ -0,0 +1,19 @@
+[
+  (class_declaration)
+  (class_body)
+  (constructor_declaration)
+  (constructor_body)
+  (block)
+  (switch_block)
+  (array_initializer)
+  (argument_list)
```

Comment:
```suggestion
  (argument_list)
  (formal_parameters)
```
Indentation of formal_parameters behaves the same way (not ideal that is), but better than nothing

Reply:
@theHamsta is the cpp maintainer, can you shed light?